Gemfile

```diff
@@ -1,6 +1,5 @@
 source 'https://rubygems.org'
 
 # locked to github pages versions
-gem 'jekyll',     '=1.1.2'
-gem 'rdiscount',  '=1.6.8'
-gem 'albino'
\ No newline at end of file
+gem 'jekyll',     '=3.2.1'
+gem 'kramdown',  '=1.11.1'
```

Comment:
https://pages.github.com/versions/
https://github.com/blog/2100-github-pages-now-faster-and-simpler-with-jekyll-3-0

```
Starting May 1st, 2016, GitHub Pages will only support kramdown, Jekyll's default Markdown engine. If you are currently using Rdiscount or Redcarpet we've enabled kramdown's GitHub-flavored Markdown support by default, meaning kramdown should have all the features of the two deprecated Markdown engines, so the transition should be as simple as updating the Markdown setting to kramdown in your site's configuration (or removing it entirely) over the course of the next three months.
```